src/SchemeReport.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SPDXLicenseCollection xmlns="http://www.spdx.org/license">
+   <license isOsiApproved="false" licenseId="SchemeReport"
+            name="Scheme Language Report License">
+
+    <text>
+      <p>Code derived from the document "Revised Report on 
+        Scheme" and subsequent revisions, is distributed under the following license:
+      </p>
```

Comment:
I couldn't find this in the example projects listed in #1340; if it's not part of the license text itself, it should probably go in a `<notes>` element before the `<text>` element.